ingestion_tools/scripts/data_validation/source/test/test_tiltseries.py

```diff
@@ -0,0 +1,63 @@
+import allure
+import pandas as pd
+import pytest
+from importers.base_importer import RunImporter
+from importers.tiltseries import TiltSeriesImporter
+
+from common.fs import FileSystemApi
+from common.image import get_volume_info
+from data_validation.shared.helper.tiltseries_helper import TiltSeriesHelper
+from data_validation.shared.util import get_file_type, get_mrc_header, get_zarr_metadata
+
+
+@pytest.mark.tiltseries
+@pytest.mark.parametrize(
+    "tiltseries",
+    pytest.cryoet.tiltseries,
+    ids=[f"{ts.get_run().name}-{ts.identifier}" for ts in pytest.cryoet.tiltseries],
+    scope="session",
+)
+class TestTiltSeries(TiltSeriesHelper):
+
+    @pytest.fixture(scope="session")
+    def run(self, tiltseries: TiltSeriesImporter) -> RunImporter:
+        return tiltseries.get_run()
```

Comment:
Does this get used?

Reply:
Good catch! It does get used indirectly for generating the fixture components for mdoc, rawtlt. I will add a comment to make it more explicit. :)